### ISSUE TITLE
synchronise georss table in line with map

### ISSUE THREAD

**Issue by matthewberryman** [MSFREACH/msf-reach#669]
Do we want to have the table only show the ones that are visible in the map? I would argue no:
1. We don't want people reviewing the list and missing disasters just because of the map zoom level.
2. The table will pan and zoom the map and and focus on clicking of the table (but noting #668)

**Comment by LucieGueuning** [comment]
I insist because: 
- It will remove noisy info for people having their ountry portfolio in MENA and not in Asia (for example). 
- It is already the case for Contact and all other data layer. Keep consistency.

**Comment by anexact** [comment]
Re: @matthewberryman's point: "We don't want people reviewing the list and missing disasters just because of the map zoom level." I think this is very important point, @LucieGueuning any way we can keep the noise down but avoid that risk?

**Comment by matthewberryman** [comment]
For each of the different instantiations of REACH we could restrict the list to a fixed area relevant for that region (e.g. West Africa).

**Comment by LucieGueuning** [comment]
Yes, continuing the brainstorming: so that, people could choice which region they want for their profile. We might have people monitoring West Africa and Nairobi from Europe. They would use both regions for RSS feed then? 

Please let me know your thoughts on that.
